scripts/aggregate-themes/aggregate-themes.js

```diff
@@ -0,0 +1,150 @@
+const fs = require('fs');
+const glob = require('glob');
+const path = require('path');
+const Logger = require('../utils/logger');
+
+const CONFIG = 'theme.config.js';
+const NODE_MODULES = 'node_modules/';
+const OUTPUT = 'aggregated-themes.js';
+const OUTPUT_PATH = path.resolve(process.cwd());
+const DISCLAIMER = fs.readFileSync(path.resolve(__dirname, 'disclaimer.txt'), 'utf8');
+
+/**
+ * Aggregates theme assets into a single file.
+ *
+ * By default the theme will recursively search all dependencies.
+ * If a theme directory contains a root-theme.scss only that single file will be aggregated.
+ */
+class ThemeAggregator {
+  /**
+   * Aggregates theme assets.
+   * @returns {string|null} - The output path of the aggregated theme file. Null if not generated.
+   */
+  static aggregate() {
+    const defaultConfig = path.resolve(process.cwd(), CONFIG);
+
+    if (fs.existsSync(defaultConfig)) {
```

Comment:
Do we need to double-check this is not a directory? This is what we've done for loading i18n config and translation files

Reply:
The config file is required to be at the same level as the process being run. With the current implementation it can't be a directory. 

```
project
├── theme.config.js
├── webpack.config.js
```